.github/workflows/maven.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
         id: yaml-data
         uses: jbutcher5/read-yaml@1.6
         with:
-          file: './src/test/resources/config/spark/experiment_config_${{ matrix.lst }}.yaml'
+          file: './src/test/resources/config/spark/experiment_config-${{ matrix.lst }}.yaml'
```

Comment:
Clarification: for context, could you please explain the motivation for using a different naming convention for the table format name in files. Is it a requirement or to highlight the fact that it is picked from the yaml config. Thanks

Reply:
Yes, I just made this change because we were using different naming conventions for the files. Indeed, I wanted to create a small separation between what the file is about and the fact that it is specific for a certain LST. But as you pointed out, this is not a requirement.